onboard/src/pages/WebPortal.jsx

```diff
@@ -29,38 +29,30 @@ const menuList = [
   },
   {
     index: 2,
-    name: "Work Preference",
-    title: "Where would you PREFER to work?",
-    completed: false,
-    child: <WorkingLocationPage />,
-    visited: false,
-  },
-  {
-    index: 3,
-    name: "Computer and Accessories",
-    title: "Computer and Accessories",
+    name: "Computer & Work Preferences",
+    title: "Computer & Work Preference",
     completed: false,
     child: <EquipmentPage />,
```

Comment:
Maybe consider changing the component name as well to prevent confusion in the future